unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/BuildModeHUDController.cs

```diff
@@ -80,15 +80,40 @@ public BuildModeHUDController()
 
         view.OnTutorialAction += () => OnTutorialAction?.Invoke();
         view.OnPublishAction += () => OnPublishAction?.Invoke();
+        view.OnLogoutAction += () => OnLogoutAction?.Invoke();
+    }
+
+    public void StartPublishing()
+    {
+        view.StartPublishing();
+    }
+
+    public void PublishEnd(string message)
+    {
+        view.PublishEnd(message);
     }
```

Comment:
I see that the view has `StartPublishing` and `PublishEnd`. If both methods are referring to the same lifecycle (Publishing lifecycle) they should have the same wording structure. e.g.: `StartPublish`/`EndPublish` or `PublishStart`/`PublishEnd`, etc.